jira7/tests/tests.bal

```diff
@@ -26,23 +26,19 @@ JiraConfiguration jiraConfig = {
 
 Client jiraConnectorEP = new(jiraConfig);
 
-function formatJiraConnError(JiraConnectorError e) returns string {
-    return string `${e.message} - ${e.jiraServerErrorLog.errors.toString()}`;
-}
-
 @test:BeforeSuite
 function connector_init() {
     //To avoid test failure of 'test_createProject()', if a project already exists with the same name.
-    _ = jiraConnectorEP->deleteProject("TSTPROJECT");
+    var projectDeleted = jiraConnectorEP->deleteProject("TSTPROJECT");
```

Comment:
Let's use checkpaic here instead. We don't want the tests to run if `connector_init()` fails right?

Reply:
> We don't want the tests to run if `connector_init()` fails right?

yes

Reply:
Does that mean `jiraConnectorEP->deleteProject("TSTPROJECT");` here returns an error?

Reply:
yes it return error when we don't have project with name "TSTPROJECT" in JIRA server